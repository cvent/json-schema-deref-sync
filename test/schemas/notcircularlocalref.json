{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "properties": {
    "description": {
      "$ref": "#/description"
    }
  }
}
