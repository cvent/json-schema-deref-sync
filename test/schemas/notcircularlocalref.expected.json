{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "properties": {
    "description": "Just a basic schema."
  }
}
